src/appliance-onboarding-script/appliance_setup/pkgs/_utils.py

```diff
@@ -107,14 +107,15 @@ def create_dir_if_doesnot_exist(path: str):
     except FileExistsError:
         pass
 
-def quote_string_for_cmd(ip: str):
-    return f'"{ip}"'
+def safe_escape_characters(ip:str):
+    safe_str = ''
+    for ch in ip:
+        if ch == '"':
+            safe_str = safe_str + '\\' + ch
```

Comment:
>'\\' + ch [](http://example.com/codeflow?start=34&length=9)

Will this escaping work in both windows and linux environment?

Reply:
tested in linux and windows environment. It is working fine.